src/oaklib/implementations/sqldb/sql_implementation.py

```diff
@@ -111,14 +112,6 @@ def _mapping(m: sssom.Mapping):
     return m
 
 
-# https://stackoverflow.com/questions/16694907/download-large-file-in-python-with-requests
```

Comment:
glad to see this go.

Note to self: we have gz files on s3 now will want to download these preferentially